src/UIAutomation/UiaOperationAbstraction/UiaTypeAbstractionImpl.g.cpp

```diff
@@ -5114,6 +5114,31 @@
         return localPropertyValue;
     }
 
+    UiaVariant UiaElement::GetPropertyValue(UiaPropertyId propId, UiaBool ignoreDefault, bool useCachedApi)

```

Comment:
Add comment here specifying the default values of `ignoreDefault` and `useCachedApi`? (see `UiaElement::GetProcessId` below).